src/runtime/contrib/json/json_runtime.h

```diff
@@ -55,7 +55,7 @@ class JSONRuntimeBase : public ModuleNode {
     LoadGraph(graph_json_);
   }
 
-  const char* type_key() const { return "json"; }
+  const char* type_key() const override { return "json"; }
```

Comment:
should also be final as well, do it to other places that overrides

Reply:
done

Reply:
Actually, arm compute lib inherits from this class. I'm leaving everything as override.